luigi/contrib/rdbms.py

```diff
@@ -277,7 +277,9 @@ class Query(luigi.task.MixinNaiveBulkComplete, luigi.Task):
 
         Optionally override:
 
+        * `port`,
         * `autocommit`
+        * `update_id`
```

Comment:
I'm not able to comment on the specific line, i want. But since we're updating this docstring for other portions which should have already been included (e.g `update_id`), shouldn't we add `run` to the `Subclass and override the following methods:`?